treeherder/log_parser/tasks.py

```diff
@@ -19,8 +19,8 @@ def if_not_parsed(f):
     """Decorator that ensures that log parsing task has not already run
     """
     def inner(job_log):
-        newrelic.agent.add_custom_parameter("job_log_%i_name" % job_log.id, job_log.name)
-        newrelic.agent.add_custom_parameter("job_log_%i_url" % job_log.id, job_log.url)
+        newrelic.agent.add_custom_parameter("job_log_id", job_log.id)
```

Comment:
This will mean multiple `job_log_id` attributes are added to the same transaction, meaning earlier ones are overwritten. 

Given that the job_id is already associated with the transaction, perhaps we don't need to store the job_log_id at all? (Since there isn't much else of interest in the model, and someone can always look up the row using the job_id and JobLog name if needs be.)